index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <div class="container" id="main">
```

Comment:
It would have been better if you used a semantic HTML tag, for instance in this case, <section> instead of <div>

Reply:
Kindly explain why you suggested this comment, so the student could understand better 😄 👍

Reply:
Good work on your project so far. But we recommend using semantic HTML tags, for instance (main) instead of (div with a class main). Semantic tags make your code more meaningful and maintainable. This also helps in SEO of your website as browser can identify the meaning of a page and its content faster. Adapting the habit of using semantic tags from now, will help you a lot in the future. 😁